src/components/ComputeModal.js

```diff
@@ -1348,6 +1348,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
             existingRuntime.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp)
           ])],
           [willRequireDowntime(), () => h(Fragment, [
+            existingRuntime.tool !== tool ? p(['By continuing, you will be changing the application of your cloud environment from ', span({ style: { fontWeight: 'bold' } }, existingRuntime.tool), ' to ', span({ style: { fontWeight: 'bold' } }, tool), '.']) : undefined,
```

Comment:
For emphasizing text, we use `strong`, which would be detected by a screen reader and also would eliminate custom styling:

```suggestion
            existingRuntime.tool !== tool ? p(['By continuing, you will be changing the application of your cloud environment from ', strong([existingRuntime.tool]), ' to ', span({ style: { fontWeight: 'bold' } }, tool), '.']) : undefined,
```